widgetssdk/src/main/java/com/glia/widgets/filepreview/ui/ImagePreviewActivity.kt

```diff
@@ -69,7 +63,7 @@ internal class FilePreviewActivity : AppCompatActivity(), FilePreviewContract.Vi
         enableEdgeToEdge()
         super.onCreate(savedInstanceState)
         SimpleWindowInsetsAndAnimationHandler(binding.root, binding.toolbar)
-        Logger.i(TAG, "Create File Preview screen")
+        Logger.i(TAG, "Create Image Preview screen")
```

Comment:
It's better to leave existing message because it's aligned with iOS platform...

Reply:
But what benefits can we expect from aligning this with iOS?

I think it's better to log it in a way that matches the current Android screen.
I renamed this activity and related items because it is only for previewing images. I think it is confusing to name it FilePreview.

Also, why not just rename iOS either?

Reply:
> what benefits can we expect from aligning this with iOS?
There was a goal to have as much the same messages as possible. To be able to search in Kibana and compare iOS/Android behavior in some cases.

> I think it's better to log it in a way that matches the current Android screen.
Yes, agree. Let's ask iOS to change it?

Reply:
I will create a ticket for iOS.